modules/shared/logic/formatDate/FormatDate.ts

```diff
@@ -0,0 +1,14 @@
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
+
+dayjs.extend(relativeTime);
```

Comment:
Thanks for taking the time and select the most relative and correct approach to get the human readable duration without adding more libraries to the application. I like it👍